MultiplayerMod/Game/Tools/Events/BuildToolPriority.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace MultiplayerMod.Game.Tools.Events;
+
+public static class BuildToolPriority {
+
+    public static PrioritySetting Get() {
```

Comment:
Nit:
Why not getter?

Reply:
Why getter lmao? 
My reasoning is the class is a property itself and this `Get` is the getter.
Now I have a thought it could be placed into `GameState` right?

Reply:
Usually `GetSomething` in C# could be and should be replaced with getters as more default and native way.
Unless it is not a getter but something like computation task with result or whatever else.

Move to GameState seems good to me.

Reply:
Moved.